wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationMessageSnapshotTestCase.swift

```diff
@@ -32,7 +33,26 @@ extension ConversationMessageContext {
                                                                        spacing: 0)
 }
 
-extension XCTestCase {
+func XCTAssertArrayEqual(_ descriptions: [Any], _ expectedDescriptions: [Any], file: StaticString = #file, line: UInt = #line) {
```

Comment:
I suggest to remove this helper. I think it is already being used in a wrong way. I will create a PR, if you agree? @agisilaos

Reply:
I can take a look on how it's being used but sure, please do open a separate PR. If not please do create a ticket so we can tackle it later.